lib/cnet/sbin/cnet_ifshow.c

```diff
@@ -88,7 +88,7 @@ netif_show(struct netif *netif, char *ifname)
     cne_printf("\n");
 
     cne_printf("%9s[magenta]RXE[]:[red]%'16" PRIu64 "[] [magenta]TXE[]:[red]%'16" PRIu64
-               "[] [magenta]MIS[]:[red]%'16" PRIu64 "[] \n",
+               "[] [magenta]MISS[]:[red]%'16" PRIu64 "[] \n",
```

Comment:
The use of MIS was for alignment of the output. Please check to make sure the output is aligned.